services/QuillLMS/.rspec

```diff
@@ -1,4 +1,5 @@
 --colour
 --color
 --format progress
---format ParallelTests::RSpec::SummaryLogger --out tmp/spec_summary.log
\ No newline at end of file
+--format ParallelTests::RSpec::SummaryLogger --out tmp/spec_summary.log
+--exclude-pattern "spec/integration/*"
```

Comment:
We might want to rename this directory to something slightly different, since `integration` can be used by `rspec` for `request` specs. https://relishapp.com/rspec/rspec-rails/docs/directory-structure
> Request specs reside in the spec/requests directory. The directory can also be named integration or api.

Reply:
I would second this recommendation.  This would also keep consistency with the [CMS](https://github.com/empirical-org/Empirical-Core/tree/develop/services/QuillCMS/spec/requests)

Reply:
Another approach might be using [tags](https://relishapp.com/rspec/rspec-core/v/2-5/docs/command-line/tag-option) on specific files/tests rather than whole directories.  For example,

`describe 'ComprehensionIntegration', external_apis: :true do`

Then you can specify which tags to exclude from normal circleci runs.

Reply:
I've changed the directory name to `skip_ci` for now.  We can change it again later if we want.